packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -754,14 +762,19 @@ export class MockFluidDataStoreRuntime
 		return `/${this.id}`;
 	}
 
-	private _local = false;
-
+	/**
+	 * @deprecated Use `attachState` instead
+	 *
+	 * @privateRemarks Also remove the setter when this is removed. setters don't get their own doc tags.
+	 */
 	public get local(): boolean {
-		return this._local;
+		return this.isAttached;
```

Comment:
I see you resolved your comment here but still looks like it needs fixing in latest.